Px.Utils.UnitTests/ModelBuilderTests/MatrixMetadataBuilderTests.cs

```diff
@@ -525,5 +528,62 @@ public void MultilanguageTableWithCustomMetaPropertiesAndPropetyTypes()
             Assert.AreEqual(MetaPropertyType.TextArray, actual.AdditionalProperties["SINGLEITEMTEXTARRAYPROPERTY"].Type);
             Assert.AreEqual(MetaPropertyType.MultilanguageTextArray, actual.AdditionalProperties["SINGLEITEMMULTILANGUAGETEXTARRAYPROPERTY"].Type);
         }
+
+        [TestMethod]
+        public void MultilanguageRemovesDimensionTypeAndTimeValEntriesTest()
+        {
+            Assert.IsTrue(Actual_1Lang_With_Range_Time_Dimension.Dimensions.Exists(dim => dim.Type == DimensionType.Time));
+            foreach (Dimension dim in Actual_3Lang.Dimensions)
```

Comment:
This test is rather confusing, it uses multiple fixtures and the content doesn't match the name

Reply:
Tests fixed